cmd/krel/cmd/ff.go

```diff
@@ -28,9 +28,10 @@ import (
 )
 
 type ffOptions struct {
-	branch    string
-	masterRef string
-	org       string
+	branch         string
+	masterRef      string
+	org            string
+	nonInteractive bool
```

Comment:
I'm not very happy with this but it seems the only way to skip the interactive-ness of the `util.Ask` function.

Reply:
Yeah, on the upside, we don't expose the option on in the flags, so it should be fine.

Reply:
I actually think this is ok, I was reading this [post](https://pace.dev/blog/2020/02/12/why-you-shouldnt-use-func-main-in-golang-by-mat-ryer) recently, and I think providing this as an option is a good practice :)